edb/schema/pointers.py

```diff
@@ -1211,6 +1211,17 @@ def _parse_computable(
                     context=srcctx,
                 )
 
+        if (
+            not isinstance(source, Pointer)
+            and not source.is_view(schema)  # type: ignore
+            and target.is_view(expression.irast.schema)
+        ):
+            raise errors.UnsupportedFeatureError(
+                f'including a shape on schema-defined computed pointers '
```

Comment:
We tend to avoid using "pointer" in user-facing messages and documentation, so let's spell "properties" or "links" depending on what is being defined.

Reply:
oh, and in fact it is always a link